product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/pom.xml

```diff
@@ -37,8 +37,8 @@
         <module>01-synapseConfig</module>
         <module>01-synapseConfigCompositeApplication</module>
 
-        <!--Scenario Specific CAR Applications -->
-        <!--<module>1_8-synapse-configCompositeApplication</module>-->
+        <!--Scenario Specific CAR Applications should go here, if needs to be built-->
```

Comment:
shall we correct grammer "if need/needed to be built"

Reply:
Corrected. It is "first conditional" - according to grammar. So should be "need" not "needed".